api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -1505,6 +1559,115 @@ public CohortDefinition getAllPatientsOnSarcomaKarposi() {
     return cd;
   }
 
+  /*
+   * Get number of patients participating in at least one DSD model
+   *
+   * @return CohortDefinition
+   * */
+  public CohortDefinition getPatientsParticipatingInAtLeastOneDsdModel() {
+    SqlCohortDefinition cd = new SqlCohortDefinition();
+    cd.setName("participatingInDsdModel");
+    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
+    cd.setQuery(
+        DsdQueries.getPatientsParticipatingInDsdModel(
+            hivMetadata.getPrevencaoPositivaInicialEncounterType().getEncounterTypeId(),
```

Comment:
on my understanding it  looks like we are not  using the correct (encounter_type =6)  we are referring 34 and  35  while  the spec refers to  6

Reply:
Fixed